css/style.css

```diff
@@ -9,285 +14,616 @@ body {
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag-title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;  
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 1px;
+  opacity: .5;
 }
 
 .total-look__text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .name {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 12px;
-    line-height: 15px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
 }
 
-.new-collection__background {
-    background-size: 100% auto;
-}
-
-
-    @media (max-width: 768px) { 
-        .logo,
-        .lang,
-        .check,
-        .new-сollection__social,
-        .text,
-        .new-сollection__main-card,
-        .new-collection__background {
-            display: none;
-            } 
-        } 
-
-        @media (min-width: 768px) { 
-        .logo-mobil,
-        .galka {
-            display: none;
-            } 
-        } 
-
-        @media (max-width: 768px) { 
-            .girl {
-                background-image: none;
-            }
-        }
+/* Cтили 2й секции ниже */
+
+.bag {
+  background: #fae8e4;
+  height: 900px;
+  margin-top: 0;
+  width: 1440px;
+}
+
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
+
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px;
+}
+
+.card {
+  position: absolute;
+  width: 488px;
+  height: 556px;
+  margin-left: 879px;
+  top: 1068px;
+  background: #FFFFFF;
+}
+
+.bag__title-bold {
+  position: absolute;
+  width: 400px;
+  height: 41px;
+  left: 55px;
+  top: 40px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  color: #000000;
+ }
+
+.bag__color {
+  position: absolute;
+  top: 375px;
+  left: 55px;
+}
+
+.color-wight,
+.color-blue,
+.color-green {
+  width: 30.18px;
+  height: 30px;
+}
+
+.color-pink {
+  width: 38px;
+  height: 38px;
+}
+
+.color-wight {
+  background:#DED;
+}
+
+span {
+  position: absolute;
+  width: 125px;
+  height: 57px;
+  left: 288px;
+  top: 365px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  color: #000;
+}
+
+.button-minus,
+.button-plus {
+  border: none;
+}
+
+.button-number {
+  position: absolute;
+  width: 100px;
+  height: 44px;
+  left: 55px;
+  top: 468px;
+  border: 1.5px solid #000;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.bag__button {
+  position: absolute;
+  width: 253px;
+  height: 44px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  left: 170px;
+  top: 468px;
+  background: #FFF;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)